### PR TITLE
Fix flatpak test for aarch64

### PR DESCRIPTION
**Pull request by grisu48**
flatpak tests fail on aarch64 because they rely on the steam package,
which is not available there. This commit replaces steam with vlc.

Also increase the timeout for installing gimp, which appeared to be an issue on some VRs.

- Related ticket: https://progress.opensuse.org/issues/88582
- Verification run: [Tumbleweed aarch64](https://openqa.opensuse.org/tests/1630132) | [Tumbleweed x86_64](http://duck-norris.qam.suse.de/tests/5099#step/flatpak/40)

### CODE REVIEW

File: tests/x11/flatpak.pm [hunk 1]
```diff
@@ -32,8 +32,8 @@ sub run {
     die "flatpak list is not empty" if script_output("flatpak list") != "";
     assert_script_run('flatpak remote-add --if-not-exists flathub https://flathub.org/repo/flathub.flatpakrepo');
     assert_script_run('flatpak search gimp | grep -i "org.gimp.GIMP"');
-    assert_script_run('flatpak search steam | grep -i "com.valvesoftware.Steam"');
-    assert_script_run('flatpak install -y org.gimp.GIMP');
+    assert_script_run('flatpak search vlc | grep -i "org.videolan.VLC"');
+    assert_script_run('flatpak install -y org.gimp.GIMP', timeout => 300);
```
Comment (b10n1k) [inline tests/x11/flatpak.pm:36]: does the timeout actually required here. For the commit message i am not sure why this change happened.

Comment (grisu48) [reply tests/x11/flatpak.pm:36]: Yes it is, some VRs run into timeouts there

**Review by ggkioulis — APPROVED** [review]
No steam in aarch64? Something needs to be done!
LGTM!

**Review by pdostal — APPROVED** [review]
Ready to go 🚀
